src/pages/validator/external-chains/blast.mdx

```diff
@@ -0,0 +1,185 @@
+# Blast
+
+Instructions to set up your blast node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 8GB RAM, 512GB of SSD free storage space.
+- MacOS or Ubuntu 22.04 (tested on 22.04)
+- [Official Github Documentation](https://github.com/blast-io/deployment)
+- [Official Documentation for Node Providers](https://docs.blast.io/tools/node-providers)
+
+## Prerequisites
+
+- [Common Docker Compose Commands](./optimism#common-node-operations)
+```bash
+sudo apt-get install jq -y
+snap install docker
+```
+
+## Clone Repo
+
+```bash
+git clone https://github.com/blast-io/deployment.git blast-node
+cd blast-node
+```
+
+## Set Node Config Environment Variables
+```bash
+cp .env.example .env
+```
+
+Set the following config options in `blast-node/.env.testnet` or `blast-node/.env`:
+```bash
+# NETWORK should be mainnet or sepolia
+NETWORK=mainnet
```

Comment:
We should split mainnet and testnet into tabs like in optimism.mdx

Reply:
Will add tabs